site/_docs/troubleshooting.md

```diff
@@ -209,6 +210,13 @@ strange errors where references don't exist or a tag hasn't been closed. If you
 run into these errors, try setting `excerpt_separator: ""` in your
 `_config.yml`, or set it to some nonsense string.
 
+## Production Problems
+
+If you run into an issue that a static file can't be found in your
+production environment during build since v3.2.0 you should set your
+[environment to `production`](https://jekyllrb.com/docs/configuration/#specifying-a-jekyll-environment-at-build-time).
```

Comment:
I agree with removing the domain but I don't get the logic behind a relative link? Unless Jekyll has some magic that makes these absolute they should be absolute, having links relative makes it hard to change the name of a page and while it might be rare it can happen and this makes that move more expensive than it ever should be.